Dockerfile

```diff
@@ -129,6 +132,8 @@ RUN curl -L -o $ROOTFS/usr/local/bin/generate_cert https://github.com/SvenDowide
 # Build VBox guest additions
 # For future reference, we have to use x86 versions of several of these bits because TCL doesn't support ELFCLASS64
 # (... and we can't use VBoxControl or VBoxService at all because of this)
+# TEST removing the 32 bits version
+# TODO Not working yet
```

Comment:
I wonder if we're just missing some libraries in our final ISO or something similar.